### PR TITLE
Dynamic Workers Windows 2022 updates 9 January 2024

### PR DESCRIPTION
**Pull request by flin-8**
[sc-60929]
[sc-61983]
To be merged alongside the removal of Windows 2019 EOL on 9 January 2024.

Goals:
* advise that Windows 2019 is no longer available as of 9 January 2024
* continue linking to the full timeline so customers who may have missed the notifications can see the sequence of events leading to Windows 2019 EOL

Based on https://github.com/OctopusDeploy/docs/pull/1754

### CODE REVIEW

File: src/pages/docs/infrastructure/workers/dynamic-worker-pools.md [hunk 1]
```diff
@@ -147,6 +116,12 @@ Windows 2022 workers are capable of running [execution worker containers](/docs/
 We recommend execution containers as the preferred option for steps requiring external tools. This allows you to control which version of the tools will be used as your scripts will rely on a specific version that they are compatible with to function correctly.
 :::
 
+### Windows Server Core 2019
```
Comment (flin-8) [inline src/pages/docs/infrastructure/workers/dynamic-worker-pools.md:119]: Moved 2019 below 2022

File: src/pages/docs/infrastructure/workers/dynamic-worker-pools/windows-2019-end-of-life.md [hunk 2]
```diff
@@ -9,11 +9,15 @@ hideInThisSection: true
...
+:::div{.warning}
+Windows Server 2019 images are no longer available as of 9 January 2024. The details below are provided for historical reference.
+:::
+
 Our Windows Server 2019 Dynamic Workers are being upgraded to use Windows Server 2022, this may result in breaking changes for users of community steps or custom scripts.
 
 
 ## What is changing?
-Due to LTSC Windows Server 2019 ending on 9th January 2024 , we are upgrading our dynamic workers to use Windows Server 2022.  
+Due to LTSC Windows Server 2019 ending on 9 January 2024 , we are upgrading our dynamic workers to use Windows Server 2022.
```
Comment (flin-8) [inline src/pages/docs/infrastructure/workers/dynamic-worker-pools/windows-2019-end-of-life.md:20]: Removing the `th` to make this more consistent

**Comment by steve-fenton-octopus** [comment]
We use US spellings, so you'll need to update:

utilising -> utilizing

**Comment by steve-fenton-octopus** [comment]
Spellcheck is now successful.

**Review by sahanocto — APPROVED** [review]

**Pull request merged**